cli/bench/http.rs

```diff
@@ -109,6 +109,8 @@ fn run(
   println!("{}", wrk_cmd.join(" "));
   let output = test_util::run_collect(wrk_cmd, None, None, None, true).0;
 
+  std::thread::sleep(Duration::from_secs(1)); // wait to capture failure. TODO racy.
```

Comment:
What's the reason for adding this timeout?

Reply:
Currently most of the benches actually fail after wrk ends. But this failure is not noticed by rust (because its racy).
The delay is added to give the benchscript some time to fail, so this can be noticed by rust and the ci fails too.

Reply:
Can this problem be related to https://github.com/denoland/deno/issues/9050?

Reply:
I'm not sure, normally https://github.com/denoland/deno/blob/0a32fa459d8c4398e0d4976cc5e13f1eb2791238/cli/bench/http.rs#L120 should still have killed the server.